File: java/Hardware.java
public class Hardware extends Product{
    
    private String brand;
    private String model;

    public Hardware(){
    }

    public Hardware(String b, String m){
        this.brand = b;
        this.model = m;
    }
    
    public String getBrand() {
        return brand;
    }
    
    public String getModel() {
        return model;
    }
    
    public void setModel(String model) {
        this.model = model;
    }
    
    public void setBrand(String brand) {
        this.brand = brand;
    }

    public void showHardware(){
        System.out.println("Brand         : " + brand);
        System.out.println("Model         : " + model);
    }

    
}

File: python/Hardware.py
from Product import Product

class Hardware(Product):

    #konstruktor
    def __init__(self):
        self.brand = ""
        self.model = ""

    #membuat setter dan getter
    def setBrand(self, brand):
        self.brand = brand
    
    def getBrand(self):
        return self.brand

    def setModel(self, model):
        self.model = model
    
    def getModel(self):
        return self.model

    # Menampilkan isi kelas
    def showHardware(self):
        print("Brand         : " + str(self.brand))
        print("Model         : " + str(self.model))